app/controllers/guest_rooms_controller.rb

```diff
@@ -0,0 +1,11 @@
+class GuestRoomsController < ApplicationController
+  def create
+    guest = Guest.find(params[:id])
+    room = Room.find(params[:room_id])
+
+    guest.rooms << room
+    guest.save
+
+    redirect_to "/guests/#{guest.id}" 
+  end
+end
```

Comment:
This looks good. One thing I really like about this class is how you've created your own controller for creation of a GuestRoom. 

Although your code here is clean and will work, I think there might be a more optimal way to do this. Currently, you are finding the Guest object AND the Room object, and then creating that connection between the guest and room. Instead, you have EVERYTHING you need right away to create that connection. You could have something like this to make your code more optimal:

```ruby
def create 
  GuestRoom.create(room_id: params[:room_id], guest_id: params[:id])
  redirect_to "/guests/#{params[:id]}"
end 
```